modules/microsoft.cdn/azure-frontdoor-cdn-profile/1.0/main.bicep

```diff
@@ -0,0 +1,80 @@
+@sys.description('Name of Azure CDN SKU')
+param skuName string
```

Comment:
add `allowed` and a default values for each param (where practical), esp for the object arrays so it's easier to identify the schema

Reply:
Fixed.